pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>com-skynest-uitest</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <testng.version>7.6.0</testng.version>
+        <selenium.version>4.1.3</selenium.version>
+        <webdrivermanager.version>5.1.1</webdrivermanager.version>
+        <javafaker.version>1.0.2</javafaker.version>
+        <projectlombok.version>1.18.24</projectlombok.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.testng</groupId>
```

Comment:
Defining a scoped dependency alongside its non-scoped variant is redundant, so you might as well remove this one

Reply:
Removed